client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -101,6 +101,18 @@ class PublishMenu extends PureComponent {
 				showOnAllMySites: false,
 			} );
 		}
+
+		items.push( {
+			name: 'discussions',
+			label: this.props.translate( 'Discussions' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **100** times:
`translate( 'Discussion' )` *ES Score: 19*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/sidebar-menu-discussions)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).